ibis/expr/api.py

```diff
@@ -218,7 +218,8 @@
 
 Instead, use the typed `ibis.null`:
 
->>> ibis.null(str).upper().execute() is None
+>>> ibis.options.interactive = True
```

Comment:
Do we need to set this again down here, or is setting it above sufficient?

Reply:
Oh, good point, I think it should be sufficient to set it once!